scripts/resize.py

```diff
@@ -93,11 +93,11 @@ def process(self, inputs, outputs):
         log.ODM_DEBUG('Resizing dataset to: %s' % tree.dataset_resize)
 
         # check if we rerun cell or not
-        rerun_cell = (args.rerun is not None and
-                      args.rerun == 'resize') or \
-                     (args.rerun_all) or \
-                     (args.rerun_from is not None and
-                      'resize' in args.rerun_from)
+        rerun_cell = (args['rerun'] is not None and
```

Comment:
I'm seeing the lines:

```
(args['rerun'] is not None and
 -                      args.rerun == 'resize') or \		 +                      args['rerun'] == 'resize') or \
 -                     (args.rerun_all) or \		 +                     (args['rerun_all']) or \
 -                     (args.rerun_from is not None and		 +                     (args['rerun_from'] is not None and
``` 

Being used in several places, this would be a good place to add the logic for checking the rerun in a function. Doesn't have to be done as part of this PR, but it's an improvement for the future.